FirebaseVertexAI/Sources/GenerateContentResponse.swift

```diff
@@ -88,6 +88,26 @@ public struct GenerateContentResponse: Sendable {
     }
   }
 
+  /// Returns inline data parts found in any `Part`s of the first candidate of the response, if any.
+  public var inlineDataParts: [InlineDataPart] {
+    guard let candidate = candidates.first else {
+      VertexLog.error(
+        code: .generateContentResponseNoCandidates,
+        "Could not get inline data parts because the response has no candidates. The accessor only checks the first candidate."
+      )
+      return []
+    }
+    let inlineDataParts: [InlineDataPart] = candidate.content.parts.compactMap { part in
+      switch part {
+      case let inlineDataPart as InlineDataPart:
+        return inlineDataPart
+      default:
+        return nil
+      }
+    }
+    return inlineDataParts
```

Comment:
nit: Consider using `as?` instead of a `switch`.
```suggestion
    return candidate.content.parts.compactMap { $0 as? InlineDataPart }
```